api-reference/10 UI Components/Errors and Warnings/W1016.md

```diff
@@ -94,5 +93,4 @@ Use the **name** and **acceptedValues** properties instead:
         </Toolbar>
     </HtmlEditor>
 
-
 ---
```

Comment:
This file should be fixed in 21.1 too.

Reply:
And it will be fixed https://github.com/DevExpress/devextreme-documentation/pull/2302/files#diff-22a24b4536541f2e271e8f8cfba8d7cd63fd32bb9acf12dccfb29a0a42022189